.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+orbs:
+  codecov: codecov/codecov@1.0.2 # to get this to work, had to opt-in to using third party orbs in Organization Security settings.
+jobs:
+  puget:
+    parameters:
+      python_version:
+        type: string
+    docker:
+      - image: continuumio/miniconda:latest
+    environment:
+      PYTHON: << parameters.python_version >>
+      ENV_NAME: puget_tests
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps-{{ .Branch }}-{{ checksum ".circleci/puget_tests.yml" }}
+      - run:
+          name: configure conda environment
+          command: ./.circleci/install-circle.sh
+      - run:
+          name: install
+          command: |
+            source activate ${ENV_NAME}
+            pip install recordlinkage
```

Comment:
Doesn't this one get picked up by the `pip install .`?

Reply:
It probably does, I was being explicit. I can take it out.

Reply:
Probably best, so you don't end up wondering about this down the line.